SQLite.swift.podspec

```diff
@@ -1,17 +1,32 @@
-$LOAD_PATH << '.'
-require 'Supporting Files/podspec.rb'
+#
+# `pod lib lint SQLite.swift.podspec' fails - see
```

Comment:
Hm, I'm getting this:

```
 -> SQLite.swift (0.9.0)
    - ERROR | xcodebuild: Returned an unsuccessful exit code.
    - ERROR | [iOS] swift: Swift support uses dynamic frameworks and is therefore only supported on iOS > 8.
    - ERROR | xcodebuild:  /Users/stephencelis/Developer/SQLite.swift/SQLite/Core/Blob.swift:25:8: error: no such module 'SQLite3'

[!] SQLite.swift did not pass validation, due to 3 errors.
```

I can eliminate the middle one by adding this line:

``` rb
  s.ios.deployment_target = "8.0"
```

The other two, when running `--verbose`, are failing on the `import SQLite3`, assuming because there's no `0.9.0` tag with the `SQLite3` directory/sub-pod available?